scripts/backtest_sites.py

```diff
@@ -412,7 +477,13 @@ def main(config: DictConfig):
     # Create a dataloader for the concurrent batches and use multiprocessing
     dataloader = DataLoader(batch_pipe, **dataloader_kwargs)
     # Load the PVNet model
-    model, *_ = get_model_from_checkpoints([model_chckpoint_dir], val_best=True)
+    if model_chckpoint_dir is not None:
```

Comment:
I think this could be simplified to 
```suggestion
    if model_chckpoint_dir:
```
and similarly the `elif` below